doc/mgr/orchestrator.rst

```diff
@@ -152,23 +157,21 @@ Create OSDs on a group of devices on a single host::
 
     ceph orch daemon add osd <host>:device1,device2
 
-or::
-
-    ceph orch apply osd -i <json_file/yaml_file>
+Example::
 
+    # ceph orch daemon add osd node1:/dev/vdd
+    Created 1 OSD on host 'node1' using device '/dev/vdd'
 
 or::
 
-    ceph orch apply osd --all-available-devices
-
+    ceph orch apply osd -i <json_file/yaml_file>
 
+Being <json_file/yaml_file> a drive group specification-
```

Comment:
s/  Being <json_file/yaml_file> a drive group specification/ Where <json_file/yaml_file> is a drive-group specification/